daemon/daemon.go

```diff
@@ -367,12 +368,36 @@ func (d *Daemon) compileBase() error {
 	return nil
 }
 
+func (d *Daemon) registerK8sAddresses(k8sNode *v1.Node, v4CIDR, v6CIDR *net.IPNet) {
```

Comment:
We should really move these things to `pkg/k8s` instead of continuing to extend the daemon.